napari/_qt/qt_main_window.py

```diff
@@ -129,6 +132,13 @@ def _add_window_menu(self):
         self.window_menu = self.main_menu.addMenu('&Window')
         self.window_menu.addAction(exit_action)
 
+    def _add_help_menu(self):
+        about_action = QAction("napari info", self._qt_window)
```

Comment:
can this be "about napari" instead of "napari info"

Reply:
that was my initial intention. On mac osx with native menubar 'about' word in action name prevents creation of help menu. That's why i changed to napari info, if you can find a way to make it work i will be happy to change.